h/static/scripts/cross-frame.coffee

```diff
@@ -46,11 +46,24 @@ module.exports = class CrossFrame
         if err
           channel.destroy()
         else
+          searchUris = [info.uri]
+
           documentFingerprint = null
           if info.metadata and info.metadata.documentFingerprint
             documentFingerprint = info.metadata.documentFingerprint
+
+            reducer = (uris, link) ->
+              uris.push(link['href'])
+              return uris
+            searchUris = info.metadata.link.reduce(reducer, []);
```

Comment:
If you have an array of values that you want to produce an equal-length array of some derived value, you can use `Array#map` instead. eg. `info.metadata.link.map((link) -> link.href)`